provider/status_changes.json

```diff
@@ -363,19 +292,103 @@
                 "minimum": 0,
                 "maximum": 1
               },
-              "associated_trips": {
-                "$id": "#/properties/data/properties/status_changes/items/properties/associated_trips",
-                "type": [
-                  "array",
-                  "null"
-                ],
-                "description": "For 'Reserved' event types, associated trip_ids",
-                "items": {
-                  "$id": "#/properties/data/properties/status_changes/items/properties/associated_trips/items",
-                  "$ref": "#/definitions/uuid"
-                }
+              "associated_trip": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/associated_trip",
+                "description": "Trip UUID, required if event_type_reason is user_pick_up or user_drop_off",
+                "$ref": "#/definitions/uuid"
               }
-            }
+            },
+            "allOf": [
+              {
+                "description": "valid event_type and event_type_reason combinations",
+                "oneOf": [
+                  {
+                    "properties": {
+                      "event_type": {
+                        "enum": [
+                          "available"
+                        ]
+                      },
+                      "event_type_reason": {
```

Comment:
@oderby I may be reading this wrong, but shouldn't the only `event_type_reasons` be `user_pick_up` and `user_drop_off`

Reply:
Yeah - that's specified at [line 375](https://github.com/CityOfLosAngeles/mobility-data-specification/pull/217/files#diff-f4b85af6fa965f8d99ba6dceda9e1d80R375) for the sub-schema restricting when `associated_trip` is required. Here, this is the sub-schema for valid combinations of `event_type` and `event_type_reason`